nnpdf_data/nnpdf_data/new_commondata/ATLAS_DY_13TEV/kinematics_TOT.yaml

```diff
@@ -1,37 +1,37 @@
 bins:
-- k1:
+- zero:
     min: null
     mid: 0.0
     max: null
-  k2:
+  MWZ:
```

Comment:
Please use the names from https://github.com/NNPDF/nnpdf/blob/37f48752cc362291c06df2f3990f815276bf12ee/validphys2/src/validphys/process_options.py#L16

(so `m_W2` instead of `MW2`...or... in this case, `m_V` and we will need to add a new variable for that)

Reply:
what s the procedure to add a new variable? In the case above we need `m_V`, for `ATLAS_PH_13TEV` we need the transverese ewnergy square `ET2` and I guess there will be other cases. Is it enought to add these new variables to the `_Vars` class?

Reply:
Yes. And then for the processes for which you want to add it, include it in the "accepted variables".

(if the process still doesn't exist you can either create it or leave it for later, it should not matter)